### DISCUSSION TITLE
Run a command on "cmi alert"

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#7862] [Enhancement]
Trying to think why someone would want to use /cmi alert, the way I would want this to work is, to allow me to run a command, so I could then send a message to discord with DiscordSRV command as a global command thats sent.

What would also be nice is, being able to set a command to run, for that specific add.

For example, if a command you need to run on a player only works if they are online, then this could be used to run a specific command for that player when they next login automatically using this system, so where you have a <reason> we could have a <command> and it could even use your extended command parameters to check things too! Then it gets really powerful.

**Comment by TheBadshah** [comment]
it would be great if placeholders can also be added for it so all of the alerts of that specific player can also be listed
